### PR TITLE
release studio 3.9.0

### PR DESCRIPTION
**Pull request by cooper-lzy**
https://github.com/vesoft-inc/nebula-studio/releases/tag/v3.9.0

https://github.com/vesoft-inc/nebula-ent-docs/blob/v3.7.1/docs-2.0-zh/20.appendix/release-notes/explorer-release-note.md?plain=1

### CODE REVIEW

File: docs-2.0-zh/nebula-studio/quick-start/st-ug-console.md
Comment (yyh0808) [inline docs-2.0-zh/nebula-studio/quick-start/st-ug-console.md:1]: 截屏2024-01-31 10 43 10
2这里貌似状态不对，没有全是红色的状态，最好换个截图，有颜色逻辑

Comment (cooper-lzy) [reply docs-2.0-zh/nebula-studio/quick-start/st-ug-console.md:1]: 我试了下其他语句，都是红色啊，这个排名是怎么个逻辑？
image

Comment (yyh0808) [inline docs-2.0-zh/nebula-studio/quick-start/st-ug-console.md:1]: AI助手默认是一个问答功能，然后才是自然语言转nGQL查询语句，这块需要提下text2match的switch

File: docs-2.0-en/nebula-studio/quick-start/st-ug-console.md [hunk 1]
```diff
@@ -2,24 +2,48 @@
...
-|  5  |  clean input box  | Click ![clean](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-clear.png) button to clear the content entered in the input box.  |
-|  6  |  run   |  After inputting the nGQL statement in the input box, click ![run](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-play.png) button to indicate the operation to start running the statement.  |
-|  7  |  custom parameters display   | Click the ![Query](https://docs-cdn.nebula-graph.com.cn/figures/down.png) button to expand the custom parameters for parameterized query. For details, see [Manage parameters](../../nebula-console.md).|
-|  8  |  input box   |  After inputting the nGQL statements, click the ![run](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-play.png) button to run the statement. You can input multiple statements and run them at the same time by using the separator `;`, and also use the symbol `//` to add comments. |
+|  1  |  view the schema | Display the schemas of the graph spaces. |
```
Comment (abby-cyber) [inline docs-2.0-en/nebula-studio/quick-start/st-ug-console.md:17]: Capitalize the first letter of the first word please

File: docs-2.0-en/nebula-studio/quick-start/st-ug-console.md [hunk 2]
```diff
@@ -2,24 +2,48 @@
...
-|  13  |  close execution results | Click the ![close](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-close.png) button to close the result returned by this nGQL statement. |
-|  14  |  **Table** window | Display the result from running nGQL statement. If the statement returns results, the window displays the results in a table. |
-|  15  |  **Graph** window | Display the result from running nGQL statement. If the statement returns the complete vertex-edge result, the window displays the result as a graph . Click the ![expand](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-back.png) button on the right to view the overview panel. |
+|  1  |  View the schema | Display the schemas of the graph spaces. |
+|  2  |  Select a space  | Select a space in the Current Graph Space list. The console does not support using the `USE <space_name>` statement to switch graph spaces. |
+|  3  | Favorites | Click the ![save](https://docs-cdn.nebula-graph.com.cn/figures/studio-btn-save.png) button to expand the favorites, click one of the statements, and the statement will be automatically entered in the input box. |
```
[the remark this answers is not in the document]
Comment (abby-cyber) [reply docs-2.0-en/nebula-studio/quick-start/st-ug-console.md:19]: Use comma to seperate two sentences.
And use the simple present tense for operations as we discusse.

**Review by abby-cyber — APPROVED** [review]

**Pull request merged**